src/utils.ts

```diff
@@ -0,0 +1,61 @@
+import semver from 'semver';
+
+export function isPrerelease(version: string): boolean {
+  return version.includes('-');
+}
+
+function isPreMajor(version: string): boolean {
```

Comment:
**polish**: could we add some comments to describe what a pre major is?

Reply:
2e1834a4ad7a243da15215041824b0d010732548